### DISCUSSION TITLE
When would the next Arctic Code Archive project?

### DISCUSSION THREAD

**Discussion by skadomsky** [community/community#31910] [Other Feature Feedback, Questions, & Ideas] [answered]
GitHub [made this project](https://archiveprogram.github.com/arctic-vault/) in 2020 and [also go Mars](https://github.blog/2021-04-19-open-source-goes-to-mars/) that year. Will there be next project or snapshot taken like this? Or is there any way to get these achievement badge? Thanks for answering.

**Comment by airtower-luna** [comment, marked as answer]
> Will there be next project or snapshot taken like this?

There hasn't been any announcement, so we don't know.

> Or is there any way to get these achievement badge?

No. Those are both about things made by a specific point in time, which is in the past.
